mozilla/blurts-server
# Remove workaround for incomplete jest-axe typings

`.toHaveNoViolations()` wasn't properly typed when `expect` was imported from `@jest/globals`, but our upstream fix was merged, so the workaround is no longer necessary:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/67495

# How to test

Run `npm install`, then check out a test that references `.toHaveNoViolations`. There should be no red squiggly below it. Additionally, testing-library's assertions (e.g. `.toBeInTheDocument`) shouldn't have them either.

# Checklist (Definition of Done)
